zos_concepts/gdg_datasets/copy_edit_fetch/copy_edit_fetch.yml

```diff
@@ -0,0 +1,154 @@
+
+    - set_fact:
+        tmp_data_set1: "{{ tmp_ds1.stdout }}"
+        tmp_data_set2: "{{ tmp_ds2.stdout }}"
+        date: "{{ date_time.stdout[0:10] }}"
+        time: "{{ date_time.stdout[10:] }}"
+        sys_info: "{{ sys_info.stdout | from_json}}"
+        log_output: "{{ log_output.stdout }}"
+
+    - set_fact:
+        sys_name: "{{ sys_info.data.sys_info.sys_name }}"
+    # ##########################################################################
+    # Modules: zos_data_set, zos_blockinfile, zos_copy,
+    # zos_lineinfile, zos_fetch.
+    # ##########################################################################
+    # +-------------------------------------------------------------------------
+    # | - Using zos_data_set, create two generation data groups with names
+    # |   {{ tmp_data_set1 }} and {{ tmp_data_set2 }}
+    # | - Using zos_data_set, create a generation of the GDG from {{ tmp_data_set1 }}.
+    # | - Using zos_data_set, create a generation of the GDG from {{ tmp_data_set2 }}.
+    # | - Using zos_blockinfile, add the logs to the latest generation of GDG {{ tmp_data_set1 }}.
+    # | - Using zos_copy, copy the content from the latest generation to {{ tmp_data_set1 }}
+    # | - latest generation of {{ tmp_data_set2 }}.
+    # | - Using zos_lineinfile, add a comment to a latest generation of {{ tmp_data_set2 }}
+    # |   using relative notation notation {{ tmp_data_set2 }}(0).
+    # | - Using zos_fetch, to fetch the latest generation of the GDG {{ tmp_data_set2 }}.
+    # +-------------------------------------------------------------------------
+    - name: "Create the Generation Data Group(GDG) {{tmp_data_set1}} with a limit of 2 generations"
+      zos_data_set:
+        name: "{{ tmp_data_set1 }}"
+        type: gdg
+        limit: 2
+      register: result_create_gdg
+
+    - name: "Create the Generation Data Group(GDG) {{tmp_data_set2}} with a limit of 2 generations to backup {{ tmp1 }}"
+      zos_data_set:
+        name: "{{ tmp_data_set2 }}"
+        type: gdg
+        limit: 2
+      register: result_create_gdg
+
+    - name: "Response of the GDG creation {{ tmp_data_set2 }} with a limit of 2 generations"
+      debug:
+        msg:
+          - "{{ result_create_gdg }}"
+
+    - name: "Create the GDS for {{ tmp_data_set1 }} of sequential type and record length of 200"
+      zos_data_set:
+        name: "{{ tmp_data_set1 }}(+1)"
+        type: seq
+        record_length: 200
+        state: "present"
+
+    - name: "Create the GDS for {{ tmp_data_set2 }} of sequential type and record length of 200"
+      zos_data_set:
+        name: "{{ tmp_data_set2 }}(+1)"
+        type: seq
+        record_length: 200
+        state: "present"
+      register: result_create_gds
+
+    - name: "Response of the GDS {{ tmp_data_set2 }}(+1) creation"
+      debug:
+        msg:
+          - "{{ result_create_gds }}"
+
+    - name: "Write the last 4 minutes of the operator console log into {{ tmp1 }}"
```

Comment:
{{ tmp1 }} is not a variable defined in this playbook, merely a placeholder used in my earlier suggestion